src/shared/util.js

```diff
@@ -1433,13 +1433,19 @@ MessageHandler.prototype = {
     let targetName = data.sourceName;
     let capability = createPromiseCapability();
 
-    let sendStreamRequest = ({ stream, chunk, success, reason, }) => {
+    let sendStreamRequest = ({ stream, chunk, transfers,
+                               success, reason, }) => {
+      if (transfers) {
```

Comment:
I have to ask: Why isn't is sufficient to simply always call
```javascript
this.postMessage({ sourceName, targetName, stream, streamId,
                   chunk, success, reason, }, transfers);
```
and just let the `postMessage` method deal with it?
Also, don't you need to add similar code in `_processStreamMessage` below as well?